front/src/layouts/layout-menu/layout-menu.layout.vue

```diff
@@ -0,0 +1,7 @@
+<template src="./layout-menu.layout.html"></template>
+
+<script src="./layout-menu.layout.js"></script>
+
+<style scoped src="./layout-menu.layout.css"></style>
+
+<style src="../../styles/common.css"></style>
```

Comment:
I don\t understand why you add common css in each view, if we included it in root component (app.vue)